server/middleware/validation.js

```diff
@@ -1,6 +1,6 @@
 const { FormSubmissionError, ValidationError } = require('../models/error');
 const { DOCUMENT_WHITELIST, DOCUMENT_BULK_LIMIT, VALID_DAYS_RANGE } = require('../config').forContext('server');
-const YEAR_RANGE = 120;
+const { YEAR_RANGE } = require('../libs/dateHelpers');
 const MIN_ALLOWABLE_YEAR = (new Date().getFullYear() - YEAR_RANGE);
```

Comment:
Work also moving these to a the dateHelpers? Feels like we are reusing these in multiple places too.

Reply:
Yeah might aswell, I've put the min and max years into the helper and just expose them now